src/Components/Form/Form.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC, useState } from 'react'
+import './Form.css'
+interface SearchCityProps {
+  onSearch: (cityInput: string) => void;
+}
+
+export const Form: FC<SearchCityProps> = ({onSearch}) => {
```

Comment:
As we learn more about TypeScript, I'm interested to explore if React.FC is still widely used. I've seen a couple places where it may not be necessary to have/might be becoming deprecated - might be something to look at?